src/clients/ntp-client_proposal.rb

```diff
@@ -210,21 +210,33 @@ def ui_init(rp, first_time)
       rp = deep_copy(rp)
       cont = VBox(
         VSpacing(0.5),
+        HBox(
+          HSpacing(3),
+          HWeight(
+            1,
+            Left(
+              ComboBox(
+                Id(:ntp_address),
+                Opt(:editable, :hstretch),
+                # combo box label
+                _("&NTP Server Address")
+              )
+            )
+          ),
+          HWeight(
+            1,
+            VBox(
+              UI.TextMode ? VSpacing(1) : Label(""),
```

Comment:
Please add a comment, in theory `VSpacing(1)` and `Label("")` should look the same, this condition looks a bit strange.